packages/expo/scripts/aggregate_xcprivacy/aggregate_xcprivacy.rb

```diff
@@ -0,0 +1,91 @@
+require_relative './plist/lib/plist'
+
+def get_used_required_reason_apis(installer)
+  # A dictionary with keys of type string (NSPrivacyAccessedAPIType) and values of type string[] (NSPrivacyAccessedAPITypeReasons[])
+  used_apis = {}
+  puts "Reading xcprivacy files to aggregate all used Required Reason APIs."
+  installer.pod_targets.each do |pod_target|
+    pod_target.file_accessors.each do |file_accessor|
+      file_accessor.resource_bundles.each do |bundle_name, bundle_files|
+        bundle_files.each do |file_path|
+          # This needs to be named like that due to apple requirements
+          if File.basename(file_path) == 'PrivacyInfo.xcprivacy'
+            content = Plist.parse_xml(file_path)
```

Comment:
you can read a plist with [Xcodeproj::Plist](https://github.com/CocoaPods/Xcodeproj/blob/master/lib/xcodeproj/plist.rb). 

You should be able to do `Xcodeproj::Plist.read_from_path` and get back a ruby `Hash`, which is basically an object.